tests/pkg/utils/mco_oba.go

```diff
@@ -31,6 +31,19 @@ func CheckOBAStatus(opt TestOptions, namespace, status string) error {
 	}
 }
 
+func CheckOBADeleted(opt TestOptions, namespace string) error {
+	dynClient := NewKubeClientDynamic(
+		opt.HubCluster.ClusterServerURL,
+		opt.KubeConfig,
+		opt.HubCluster.KubeContext)
+
+	_, err := dynClient.Resource(NewMCOAddonGVR()).Namespace(namespace).Get(context.TODO(), "observability-addon", metav1.GetOptions{})
+	if err == nil {
```

Comment:
better to check error status/message, only return nil when error is notfound one

Reply:
The code is changed to specifically check for `NotFound` error.